resources/Materials/TestSuite/stdlib/nodegraph_inputs/surfacematerial_nodegraph_to_surfaceshader.mtlx

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```

Comment:
Test for nodegraph with surfacematerial connected to upstream surfaceshader. Part of nodegraph connection work.